pkg/metrics/metrics.go

```diff
@@ -25,7 +25,8 @@ func Run() {
 	prometheus.MustRegister(KsmdUtilizationGV)
 
 	http.Handle(MetricPath, promhttp.Handler())
-	if err := http.ListenAndServe(MetricPort, nil); err != nil {
+	metricServer := &http.Server{Addr: MetricPort} // #nosec G112
+	if err := metricServer.ListenAndServe(); err != nil {
```

Comment:
Please help check if we can add a timeout: https://app.deepsource.com/directory/analyzers/go/issues/GO-S2112